dao-modules/src/lib.rs

```diff
@@ -1,8 +1,8 @@
-mod access_control;
-mod owner;
-mod repository;
-mod sequence;
-mod whitelist;
+pub mod access_control;
+pub mod owner;
+pub mod repository;
+pub mod sequence;
+pub mod whitelist;
```

Comment:
If we decide to make the modules public, there is no need to reexport their elements below.

Reply:
done